javascript/javascript2/week3/fastest presser.js

```diff
@@ -0,0 +1,101 @@
+const pressFaster = document.getElementById("h1");
```

Comment:
It doesn't make sense to give `id` the name that corresponds to a tag name. You probably don't need an id in this case.